tests/management/role/test_view.py

```diff
@@ -212,6 +212,30 @@ def test_create_role_allow_list_fail(self):
         response = self.create_role(role_name, in_access_data=access_data)
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
 
+    def test_create_role_appfilter_fields_fail(self):
+        """Test that we cannot create a role with an invalid operation."""
```

Comment:
Maybe `Test that we cannot create a role with an invalid key in the attributeFilter object` would be a better description? The operation value coded in the test looks valid.